tools/docker/Dockerfile

```diff
@@ -4,9 +4,13 @@ FROM ubuntu:18.04
 ENV DEBIAN_FRONTEND=noninteractive \
     DEBCONF_NONINTERACTIVE_SEEN=true
 
-# General requirements not in the base image
+# Update and upgrade.
```

Comment:
Is the upgrade bit important here? Seems like a sensible thing to do, but isn't the base ubuntu:18.04 updated regularly with the most recent packages?

Reply:
I really don't know if its upgraded regularly, but I suspect is not upgraded as much as it should... because when you run that upgrade, it actually upgrades packages.

Reply:
Alright, a bit of a drive-by, but that seems fine.

Reply:
Regarding how often they are updated: https://anchore.com/look-often-docker-images-updated/ (once or twice a month)